_opcodes/op187.html

```diff
@@ -10,15 +10,16 @@
 iwd2: 1
 pst: 0
 ---
-This effect stores a various data types, usually a variable in the 'variable' field.
-<br/>Resistance.param3: Point
-<br/>Param4: Float
-<br/>Param1: Int 
-<br/>Param2: Any
-<br/>Resource1+2+3: String
```

Comment:
This was REd, but I don't know if the engine ever cares about the type.

Reply:
> This was REd, but I don't know if the engine ever cares about the type.

I removed that part because the game only stores integer variables, so I don't know how those other parameters would even work. See [here](https://gibberlings3.github.io/iesdp/file_formats/ie_formats/gam_v2.0.htm#GAMEV2_0_Variable).

Reply:
Ok.